backend/hct_mis_api/apps/payment/signals.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any
+
+from django.core.cache import cache
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from hct_mis_api.api.caches import get_or_create_cache_key
+from hct_mis_api.apps.payment.models import PaymentPlan
+
+
+@receiver(post_save, sender=PaymentPlan)
```

Comment:
should we monitor PaymentRecord too ?

Reply:
I don't think so? As this is a version specifically for `management_payment_plans_list`, we need the status and basic info for Payment Plans for the list view. I don't think any changes to PaymentRecords impact this